apps/studio/src/contexts/EditorDrawerContext.tsx

```diff
@@ -21,24 +29,30 @@ export function EditorDrawerProvider({ children }: PropsWithChildren) {
   // Current saved state of page
   const [pageState, setPageState] = useState<IsomerComponent[]>([])
   // Current edit view of page
-  const [editorState, setEditorState] = useState<IsomerComponent[]>([])
+  const [snapshot, setSnapshot] = useState<IsomerComponent[]>([])
+  // Isomer page schema
+  const [currActiveIdx, setCurrActiveIdx] = useState(0)
 
   const value = useMemo(
```

Comment:
i don't think you need to memoize any of these. They are controlled by react. You can just return them without memoizing.

Reply:
i think it was copy paste code sadly, but sure, will remove!